assets/js/googlesitekit/notifications/register-defaults.js

```diff
@@ -569,6 +570,28 @@ export const DEFAULT_NOTIFICATIONS = {
 		},
 		isDismissible: true,
 	},
+	'module-recovery-alert': {
+		Component: ModuleRecoveryAlert,
+		priority: 320,
```

Comment:
All "alerts" such as this should be a higher priority (closer to 0) than informational notifications. Currently the zero data notification shows before this, which is a lower priority. The order should be preserved as it is today as in `BannerNotifications` (2nd only to `AdSenseAlerts`).

Reply:
I've updated it relative to the other notification of this type, I could take it closer to 0 if needed. I also left a note on the `AdSenseAlerts` to ensure this becomes higher priority and other refactors in this notification area are lower priority.